fcd/pass_argrec_remill.cpp

```diff
@@ -235,12 +235,16 @@ static std::string TrimPrefix(std::string str) {
   return ref.str();
 }
 
+template<class F>
 static void UpdateCalls(llvm::Function *old_func, llvm::Function *new_func,
```

Comment:
I think this would be better solved by just passing a const reference to `funcs` from `runOnModule()` instead of `old_func`, `new_func` and `is_old_func`.